Document/0x05a-Platform-Overview.md

```diff
@@ -367,12 +367,39 @@ An example Broadcast Receiver declaration with an intent filter in a manifest:
 </receiver>
 ```
 
+The other way is to create the receiver dynamically in code and register it with the `Context.registerReceiver()` method.
+
+An example of registering Broadcast Receiver dynamically:
+
+```Java
+// Define a broadcast receiver
+myReceiver = new BroadcastReceiver() {
+    @Override
+    public void onReceive(Context context, Intent intent) {
+        Log.d(TAG, "Intent received by myReceiver");
+    }
+};
+// Define an intent filter with actions that the broadcast receiver listens for
+IntentFilter intentFilter = new IntentFilter();
+intentFilter.addAction("com.owasp.myapplication.MY_ACTION");
+// To register the broadcast receiver
+registerReceiver(myReceiver, intentFilter);
+// To un-register the broadcast receiver
+unregisterReceiver(myReceiver);
+```
+
+Note that Android apps don’t need to be running to receive intents; the system starts apps automatically when a relevant intent is raised.
+

```

Comment:
Another thing: what does the programmatically declared receiver mean in terms of android:exported and android:enabled? How can we make sure that not everything can start this?

Reply:
I'd update both 
- https://github.com/OWASP/owasp-mstg/blob/master/Document/0x05h-Testing-Platform-Interaction.md#inspect-the-androidmanifest-2: see above comment on the default values when setting a manifest-based receiver.
- https://github.com/OWASP/owasp-mstg/blob/master/Document/0x05h-Testing-Platform-Interaction.md#inspect-the-source-code-2: add what is the default and that this has to be considered when reviewing the code.

@commjoen Do we need an extra ticket for this or better include it in this PR?

Reply:
What would you prefer @zehuanli ? would you be able to update these parts? or would you rather have it in a separate isue?

Reply:
@commjoen Yes I think I can update above parts here as they're kinda related to this issue.

Reply:
Thank you for taking care of it @zehuanli 👍